integration/tests/cook/test_basic.py

```diff
@@ -2150,42 +2073,6 @@ def get_docker_info():
             util.session.delete('%s/rawscheduler?job=%s' % (self.cook_url, job_uuid))
             mesos.dump_sandbox_files(util.session, instance, job)
 
-    def test_unscheduled_jobs(self):
```

Comment:
Relies on `util.max_cpus`, is not smart (requires creating 100 max-cpu job submissions), and there are other tests that test the `/unscheduled_jobs` endpoint